vision/src/autogluon/vision/object_detection/object_detection.py

```diff
@@ -78,37 +84,44 @@ def fit(self,
             ‘skopt’ (SKopt Bayesian optimization), ‘grid’ (grid search).
         scheduler_options : dict, default = None
             Extra options for HPO scheduler, please refer to `autogluon.core.Searcher` for details.
-        nthreads_per_trial : int, default = (# cpu cores)
+        num_cpus : int, default = (# cpu cores)
             Number of CPU threads for each trial, if `None`, will detect the # cores on current instance.
-        ngpus_per_trial : int, default = (# gpus)
+        num_gpus : int, default = (# gpus)
             Number of GPUs to use for each trial, if `None`, will detect the # gpus on current instance.
-        dist_ip_addrs : list, default = None
-            If not `None`, will spawn tasks on distributed nodes.
-        verbosity : int, default = 3
-            Controls how detailed of a summary to ouput.
-            Set <= 0 for no output printing, 1 to print just high-level summary,
-            2 to print summary and create plots, >= 3 to print all information produced during fit().
         """
-        log_level = verbosity2loglevel(verbosity)
+        log_level = verbosity2loglevel(self._verbosity)
         if self._detector is not None:
             self._detector._logger.setLevel(log_level)
             self._detector._logger.propagate = True
-            self._fit_summary = self._detector.fit(train_data, val_data, 1 - holdout_frac, random_state, resume=False)
+            self._fit_summary = self._detector.fit(train_data, tuning_data, 1 - holdout_frac, random_state, resume=False)
             return
 
         # new HPO task
+        if time_limit is None and num_trials is None:
+            raise ValueError('`time_limit` and `num_trials` can not be `None` at the same time, '
+                             'otherwise the training will not be terminated gracefully.')
         config={'log_dir': self._log_dir,
                 'num_trials': 99999 if num_trials is None else max(1, num_trials),
-                'time_limits': time_limit,
+                'time_limits': 2147483647 if time_limit is None else max(1, time_limit),
```

Comment:
It might be nicer if downstream None was handled instead of setting extremely high values. Not necessary for v0.1, more an observation.

Reply:
Also, why is config specifying 'time_limits'? Can we make it 'time_limit'?

Reply:
this internal naming and behavior change will need to be addressed in gluoncv and it may not be necessary for 0.1 release